packages/orbit-components/src/Box/index.js

```diff
@@ -0,0 +1,56 @@
+// @flow
+import * as React from "react";
+import styled, { css } from "styled-components";
+
+import defaultTheme from "../defaultTheme";
+import media from "../utils/mediaQuery";
+import { DEVICES as DEVICES_CONSTS } from "../utils/mediaQuery/consts";
+import normalize from "./normalize";
+
+import type { Props } from "./index";
+
+const StyledBox = styled(({ className, as: Element, children, dataTest }) => (
+  <Element className={className} data-test={dataTest}>
+    {children}
+  </Element>
+)).attrs(({ dataTest }) => ({ "data-test": dataTest }))`
```

Comment:
Redundant usage of `attrs`. `data-test` is being rendered directly in the return of `styled` factory.

Reply:
@vepor  it's for the test, it will not work without it